src/components/newsletter/newsletter-sign-up.tsx

```diff
@@ -126,7 +128,7 @@ const newsletterSignUpStyles = css`
     }
 
     .input-error {
-      display: block;
+      display: none;
```

Comment:
I think this undoes a change I made to hide `.input-error` using `visibility` so that they keep their slot in the grid.

Reply:
Without it, the `email` message appears underneath the `name` field when there's no `name` error. There's another way to solve this, which would be to specify the `grid-column` for each of the two error messages.